frontend/src/components/pages/your_clubs/YourClubs.tsx

```diff
@@ -84,7 +86,8 @@ export const YourClubs = () => {
    *       backend implementation.
    */
   const updateClubListAfterLeaving = async (clubId: string) => {
-    const success = await yourClubsHandlerService.leaveClub(clubId);
+    const success:boolean = await yourClubsHandlerService.leaveClub(
```

Comment:
nit: Space after the :
Or, typescript should be able to infer the type for you, so you probably don't need to specify it at all

Reply:
Good catch here! I've removed the `: boolean` and pushed the fix.